### PR TITLE
Change format of History records in DB from ISO to epoch

### PR DESCRIPTION
**Pull request by niksv**

### CODE REVIEW

File: services/src/neo4j/migrations/1.25-change-type-of-flow-event-timestamp/rollback.cql [hunk 1]
```diff
@@ -0,0 +1,3 @@
+MATCH (n:flow_event) set n.timestamp=apoc.date.toISO8601(n.timestamp)
```
Comment (sergii-iakovenko) [inline services/src/neo4j/migrations/1.25-change-type-of-flow-event-timestamp/rollback.cql:1]: I'd take precautions - like "toInteger(n.timestamp) == n.timestamp" in the "where" section.

File: src-java/kilda-persistence-neo4j/src/test/java/org/openkilda/persistence/repositories/impl/Neo4jFlowEventRepositoryTest.java [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+    private static FlowEventRepository repository;
+
+    @BeforeClass
+    public static void setUp() {
+        repository = new Neo4jFlowEventRepository(neo4jSessionFactory, txManager);
+    }
+
+    @Test
+    public void findByFlowIdAndTimeFrameOrderTest() {
```
Comment (sergii-iakovenko) [inline src-java/kilda-persistence-neo4j/src/test/java/org/openkilda/persistence/repositories/impl/Neo4jFlowEventRepositoryTest.java:50]: I'd add test for the case with 0 millisecond part of the timestamp.

Comment (niksv) [reply src-java/kilda-persistence-neo4j/src/test/java/org/openkilda/persistence/repositories/impl/Neo4jFlowEventRepositoryTest.java:50]: It's already there.  `TIME_1` is a timestamp with 0 milliseconds. On previous code version this test  fails.

File: services/src/neo4j/migrations/1.25-change-type-of-flow-event-timestamp/rollback.cql (hunk 1 above)
Comment (niksv) [reply services/src/neo4j/migrations/1.25-change-type-of-flow-event-timestamp/rollback.cql:1]: ok. will add

**Pull request merged**